### DISCUSSION TITLE
How to install fluid patcher on Mac

### DISCUSSION THREAD

**Discussion by RodMac** [GeekFunkLabs/fluidpatcher#36] [Q&A]
Hi there - I'm using & enjoying headlesspi.py on my Raspberry Pi. Now I want to install fluid patcher on my Mac so that I can edit bank files etc. I tried to follow the instructions here https://github.com/albedozero/fluidpatcher but when I enter the following command in terminal on my Mac

brew install git fluidsynth fluid-soundfont-gm python3-pip

I get this error

Warning: No available formula with the name "fluid-soundfont". Did you mean fluid-synth?
==> Searching for similarly named formulae...
This similarly named formula was found:
fluid-synth ✔
To install it, run:
  brew install fluid-synth ✔
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.

Grateful of any help to get the app installed. Thanks\

**Comment by albedozero** [comment]
I don't have a Mac to experiment on, so I was making my best guess as to which packages to install. The fluid-soundfont-gm package is needed on Linux to install the FluidR3_GM.sf2 soundfont after it was split out from the main fluidsynth package. Maybe it's still included in the homebrew package. Try either of these commands:

- using [Homebrew](https://brew.sh/):
```
brew install git fluid-synth brew-pip
```

- using [Macports](http://www.macports.org/):
```
sudo port install git fluidsynth py-pip
```

If FluidR3_GM.sf2 doesn't get installed in /usr/share/sounds/sf2, you can download a copy yourself (e.g. from [archive.org](https://archive.org/details/fluidr3-gm-gs)) and put it there, or drop it directly in SquishBox/sf2. I'm thinking about including Christian Collins's GeneralUserGS soundfont in this repo just to get around this issue...

**Reply by jik1317** [reply]
I'm also having an issue on Mac. I was able to install the library, but now when I run the pym script I get this error.

Traceback (most recent call last):
  File "/Users/-------/fluidpatcher/fluidpatcher.pyw", line 14, in <module>
    import wx, sys, webbrowser
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/wx/__init__.py", line 17, in <module>
    from wx.core import *
  File "/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages/wx/core.py", line 12, in <module>
    from ._core import *
ImportError: dynamic module does not define module export function (PyInit__core)

Just to be clear, I am supposed to run that python file right?

**Reply by jik1317** [reply]
ok fixed that issue and another one, but now getting this error

OSError: dlopen(/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages, 0x0006): tried: '/Library/Frameworks/Python.framework/Versions/3.9/lib/python3.9/site-packages' (not a file)

**Reply by jik1317** [reply]
Seems like there are some issues with openning the fluidsynth library on Mac any thoughts or solutions?
